### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by kgrase** [SadSatellites/CSC-Calendar-Project-Try2#1]
Hi! is there anyway possible I can contact you for help with this exact project.. if you could thank you soo much

**Comment by SadSatellites** [comment]
@kgrase Sure, you can either converse with me here or email me: `rm65634@email.vccs.edu`

**Comment by kgrase** [comment]
Thank you so much for replying, I had a question about the files, if I add
all the ones you've uploaded, shouldn't I run my program in the main.cpp
file?

**Comment by SadSatellites** [comment]
Yes, main.cpp is the file to run that tests the _added_ functionality of the included functions/classes. The "Final Calendar Project Try 2 Branch" has the most up-to-date files and I suggest using those files for whatever you need. The branches are iterative and represent the addition of new classes.
